main/views.py

```diff
@@ -43,29 +46,26 @@ def post(self, request, *args, **kwargs):
                                                     'registered': registered})
 
 
-class LogInView(View):
+class LogInView(TemplateView):
     Form = AuthenticationForm
-    template1 = 'login.html'
+    template_name = 'login.html'
 
     def get(self, request, *args, **kwargs):
         form = self.Form()
-        return render(request, self.template1, {'form': form})
+        return render(request, self.template_name, {'form': form})
```

Comment:
why did you need to use function get when using template view?

Reply:
i will use the get_context_data